crates/ruff_python_ast/src/nodes.rs

```diff
@@ -941,16 +987,127 @@ impl From<ExprFString> for Expr {
     }
 }
 
-/// See also [Constant](https://docs.python.org/3/library/ast.html#ast.Constant)
 #[derive(Clone, Debug, PartialEq)]
-pub struct ExprConstant {
+pub struct ExprStringLiteral {
+    pub range: TextRange,
+    pub value: String,
+    pub unicode: bool,
```

Comment:
Do we need to keep tracking the unicode flag in the AST? The only real usage i see is the pyupgrade rule that removes that prefix because it's redundant in python 3. It feels inconsistent since we don't track string flags in the AST otherwise.